tools/tune/crossover/crossover_coef_quant.m

```diff
@@ -5,7 +5,7 @@
 
 addpath ./../eq
 
-if length(lowpass) != length(highpass)
+if length(lowpass) ~= length(highpass)
```

Comment:
seems "~=" is correct, does "!=" previously wrong?

Reply:
The symbol for not equal to in MATLAB is "~="

Reply:
It appears that the author mixed up the C [ `!=`]and Matlab [ `~=`] syntaxes